tasks/release.mk

```diff
@@ -40,3 +41,6 @@ release-push-snapshot:
 	$(DOCKER) push $(app_image)
 	$(DOCKER) push $(assets_image)
 	$(DOCKER) push $(sandbox_image)
+
+## Build new images with a tag name, e.g. VERSION=auto, then push to docker hub
```

Comment:
It would be good to explain the use case for this task.  Perhaps with an example situation?